docs/test-plan.md

```diff
@@ -0,0 +1,248 @@
+* As a security conscious user, I understand the value of the password manager, what it is going to do for me without making me feel overwhelmed. I'm notified of the password manager when I'm ready to make the most use of it, and I can set up and access my accounts as easily as writing them down in a document or notebook.
+* When I log into websites and applications, I want to ensure my identity is protected, so only I can securely access my accounts with my identity.
+* When I create a new account online I want to save the newly created credentials so I can quickly log in the next time I need access to that account.
+* When I go to sign into an account, I want my account credentials to populate the form fields when available, and be accessible in one location when not.
+* When I need to change my account credentials, I want to be able to access my credentials and update them with recovery options to restore previous account credentials.
+* When I want to access my account on my mobile device, I want my account credentials accessible from when I saved them in the browser last, and vice versa, ideally without having to navigate to one location to then manually type in the credentials as viewable on screen.
+
+### Builds
+
+* Currently builds are done on an as needed basis using the installation steps in the ReadMe and docs.
+* Test Pilot build, signing and extension hosting pipeline for internal extension signing and distribution
+
+### Testing Tools
+
+* Test case creation - [TestRail](https://testrail.stage.mozaws.net/index.php?/projects/overview/51)
+* Manual test case execution - [TestRail](https://testrail.stage.mozaws.net/index.php?/projects/overview/51)
+* Bug management - [GitHub issues](https://github.com/mozilla-lockbox/lockbox-extension/issues)
+* Test automation - FoxPuppet and Selenium
+
+### Other Things
+
+* Reverting to an older version of Firefox
+* Commodity hardware that matches our target user segment
+* Connectivity requirements & user experience on a diverse range of networks
+* Ensure keyboard only use of the application function (desktop only)
+* Multi-stage workflows are logical and require as few interactions as possible.
+* When appropriate provide indications of how far through the process the user is and how much further
+* Avoidance of jargon and slang
+* Important points and content are highlighted in some way
+* User errors are clearly highlighted with help messages that suggest solutions
+
+## Sign Off
+
+### Criteria
+
+* All test cases have been executed
+
+* A sufficient level of automated test coverage has been achieved (measured by code coverage tools). Test coverage will also be measured against reduction of risk in the areas designated by the risk assessment plan.
+
+* All blockers, criticals must be fixed and verified or have an agreed-upon timeline for being fixed (as determined by engineering/QA)
+
+### Checklist
+
+<table>
```

Comment:
This works, but adding awareness that (Github flavored Markdown and) MkDocs support a more concise table definition:

http://www.mkdocs.org/user-guide/writing-your-docs/#tables

Reply:
Nice, I did not know of this thing. Done!